requirements.txt

```diff
@@ -12,3 +12,6 @@ git+https://github.com/inducer/meshpy.git
 git+https://github.com/inducer/modepy.git
 git+https://github.com/inducer/meshmode.git
 git+https://github.com/inducer/grudge.git
+
+# Toy Fortran parser for Loopy
+git+https://github.com/inducer/f2py.git
```

Comment:
@matthiasdiener Does this mean that emirge will produce a checkout of this? I'd say that's not necessary.

Reply:
Yes, it would. We could just write `f2py` instead to use pip.

Reply:
Unfortunately, `f2py` doesn't exist as a package on the package index: https://pypi.org/search/?q=f2py

To explain the context of this a bit: f2py is a Fortran->Python wrapper generator that ships with numpy. Due to limitations of the old code base (no F90), Pearu (Peterson, original author of f2py) decided to restart from scratch, starting with a Fortran parser. That's as far as he got, and a forked version of this is in that repo.

So:
- From git is the only way to install it
- But I don't think we want a checkout of it by default

-> https://github.com/illinois-ceesd/emirge/issues/58